examples/standard/deployment.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: apps/v1
-kind: Deployment
+kind: Daemonset
```

Comment:
I believe this should probably stay a deployment?

Reply:
Yes, I was testing that. I should add a TODO(revert to Deployment after testing) there.